misc/scripts/install-local.sh

```diff
@@ -55,6 +55,21 @@ function checks() {
 	if [[ -z "$hash" ]]; then
 		fancy_message warn "Package does not contain a hash"
 	fi
```

Comment:
Could we remove "Package does not contain a hash" message for `-git` pacscripts?